src/components/dialogs/error.vue

```diff
@@ -15,7 +15,7 @@
       </v-card-text>
 
       <v-card-text class="copy-normal pt-6">
-        <contact-info class="mt-2" direction="col" />
+        <ContactInfo direction="col" />
```

Comment:
VS Code colours this component specially and allows clicking F12 to go to its declaration. Example:

![image](https://user-images.githubusercontent.com/10002889/187493861-49345c43-3b6b-4031-8f3c-26568797c53c.png)